src/serac/physics/integrators/displacement_hyperelastic_integrator.hpp

```diff
@@ -40,11 +41,12 @@ class DisplacementHyperelasticIntegrator : public mfem::NonlinearFormIntegrator
    * @brief The constructor for the displacement hyperelastic integrator
    *
    * @param[in] m  HyperelasticModel that will be integrated.
+   * @param[in] thermal_m Optional thermal expansion material model
```

Comment:
Could we clarify how "optional" enabling of thermal expansion works, e.g. "if `thermal_m` is a null pointer, thermal expansion is disabled"

Alternatively, we could just provide an additional constructor
```cpp
  explicit DisplacementHyperelasticIntegrator(HyperelasticMaterial& m,
                                               GeometricNonlinearities geom_nonlin = GeometricNonlinearities::On)
       : material_(m), thermal_material_{}, geom_nonlin_(geom_nonlin)
   {
   }
```
so the user doesn't have to explicitly write `nullptr` in one of the arguments if they don't want it